src/assets/Generator.Shared/ResponseHeadersExtensions.cs

```diff
@@ -40,7 +68,16 @@ public static bool TryGetValue(this ResponseHeaders headers, string name, out Da
         {
             if (headers.TryGetValue(name, out string? stringValue))
             {
-                value = TypeFormatters.ParseDateTimeOffset(stringValue, "");
+                if (DateTimeOffset.TryParseExact(stringValue, "r", DateTimeFormatInfo.InvariantInfo, DateTimeStyles.None, out var dto) ||
```

Comment:
Is there any value in converting the `name` string to Span first?

Reply:
No, `System.Net.Http.Headers` doesn't have any span-based API